### PR TITLE
getting-started: install-fioctl: Fixing and improve token creation

### PR DESCRIPTION
**Pull request by munoz0raul**
Signed-off-by: Raul Muñoz <raul@foundries.io>

### CODE REVIEW

**Comment by doanac** [comment]
Docs for 6732d6c5c7cc1d2a2cf258f67c85079a2d88e1b1 are browesable at: https://ci.foundries.io/projects/fio-docs/builds/1067/docs/artifacts/html/index.html

File: source/getting-started/install-fioctl/index.rst [hunk 1]
```diff
@@ -131,15 +131,30 @@ credentials for interacting with Factory APIs:
...
 
-fioctl Token
-^^^^^^^^^^^^
+Application Credentials
+^^^^^^^^^^^^^^^^^^^^^^^
 
-Go to `Token <https://app.foundries.io/settings/tokens/>`_ and create a new **Api Token** by clicking on 
-the :guilabel:`+ New Token`. Complete with a **Description** and the **Expiration date** and select :guilabel:`next`.
+Go to `Token <https://app.foundries.io/settings/tokens/>`_ and create a new **Application Credentials** by clicking on 
+the :guilabel:`+ New Credentials`.
```
Comment (vanmaegima) [inline source/getting-started/install-fioctl/index.rst:148]:
```suggestion
:guilabel:`+ New Credentials`.
```

File: source/getting-started/install-fioctl/index.rst [hunk 2]
```diff
@@ -131,15 +131,30 @@ credentials for interacting with Factory APIs:
...
+
+.. figure:: /_static/install-fioctl/application_credentials.png
+   :width: 900
+   :align: center
+
+   Application Credentials
+
+
```
Comment (vanmaegima) [inline source/getting-started/install-fioctl/index.rst:156]: Can we remove this line without breaking formatting?

**Review by vanmaegima — APPROVED** [review]
minor comments, but LGTM

File: source/getting-started/install-fioctl/index.rst [hunk 3]
```diff
@@ -131,15 +131,29 @@ credentials for interacting with Factory APIs:
...
+:ref:`ref-fioctl` will now ask for your Client ID and Secret. Follow the next steps to generate it.
 
-fioctl Token
-^^^^^^^^^^^^
+Application Credentials
+^^^^^^^^^^^^^^^^^^^^^^^
 
-Go to `Token <https://app.foundries.io/settings/tokens/>`_ and create a new **Api Token** by clicking on 
-the :guilabel:`+ New Token`. Complete with a **Description** and the **Expiration date** and select :guilabel:`next`.
+Go to `Token <https://app.foundries.io/settings/tokens/>`_ and create a new **Application Credentials** by clicking on
```
Comment (doanac) [inline source/getting-started/install-fioctl/index.rst:147]:
```suggestion
Go to `Tokens <https://app.foundries.io/settings/tokens/>`_ and create a new **Application Credentials** by clicking on 
```

**Comment by doanac** [comment]
Docs for c59298cb4ffddcafba13f0b253aa1f0592033247 are browesable at: https://ci.foundries.io/projects/fio-docs/builds/1068/docs/artifacts/html/index.html

**Comment by doanac** [comment]
Docs for 28e1f521d3ffdf5b5b8aead54f1aab075d229627 are browesable at: https://ci.foundries.io/projects/fio-docs/builds/1069/docs/artifacts/html/index.html

File: source/getting-started/install-fioctl/index.rst (hunk 3 above)
Comment (munoz0raul) [reply source/getting-started/install-fioctl/index.rst]: @doanac thanks for the comment. Already fixed it.

**Review by doanac — APPROVED** [review]

**Pull request merged**